measures/ResStockArgumentsPostHPXML/measure.rb

```diff
@@ -945,16 +948,50 @@ def set_electric_panel(runner, hpxml_bldg_existing, hpxml_bldg, args)
-                          is_new_load: args[:electric_panel_load_other_new_load],
-                          component_idrefs: [])
-    end
+    branch_circuits.add(id: "BranchCircuit#{branch_circuits.size + 1}",
+                        occupied_spaces: 1,
+                        component_idrefs: [])
+    service_feeders.add(id: "ServiceFeeder#{service_feeders.size + 1}",
+                        type: HPXML::ElectricPanelLoadTypeOther,
+                        power: electric_panel_load_other_power_rating,
+                        is_new_load: args[:electric_panel_load_other_new_load],
+                        component_idrefs: [])
+  end
+
+  # Compare existence of old electric systems to new electric systems
+  def get_panel_new_loads(args, hpxml_bldg_existing, hpxml_bldg, electric_panel_load_other_power_rating)
+    return if hpxml_bldg_existing.nil?
+
+    # Heating and cooling is a bit different than the other load types. For example, if a heat pump
+    # replaces an electric furnace/AC, we wouldn't want to just assign new_load as true based on not
+    # having a heat pump in the existing building. Another example might be an electric furnace that
+    # replaces a heat pump -- we wouldn't want to call the electric furnace a new load just because
+    # it didn't have one in the existing building.
+    existing_heating_electric = (hpxml_bldg_existing.heating_systems.count { |hs| (hs.heating_system_fuel == HPXML::FuelTypeElectricity) && hs.primary_system && (hs.fraction_heat_load_served > 0) && !hs.is_shared_system } > 0) ||
+                                (hpxml_bldg_existing.heat_pumps.count { |hp| (hp.heat_pump_fuel == HPXML::FuelTypeElectricity) && (hp.fraction_heat_load_served > 0) && !hp.is_shared_system } > 0)
+
+    existing_cooling_electric = (hpxml_bldg_existing.cooling_systems.count { |cs| (cs.cooling_system_fuel == HPXML::FuelTypeElectricity) && (cs.fraction_cool_load_served > 0) && !cs.is_shared_system } > 0) ||
+                                (hpxml_bldg_existing.heat_pumps.count { |hp| (hp.heat_pump_fuel == HPXML::FuelTypeElectricity) && (hp.fraction_cool_load_served > 0) && !hp.is_shared_system } > 0)
+
+    args[:electric_panel_load_heating_system_new_load] = !existing_heating_electric && (hpxml_bldg.heating_systems.count { |hs| (hs.heating_system_fuel == HPXML::FuelTypeElectricity) && hs.primary_system && (hs.fraction_heat_load_served > 0) && !hs.is_shared_system } > 0)
+    args[:electric_panel_load_cooling_system_new_load] = !existing_cooling_electric && (hpxml_bldg.cooling_systems.count { |cs| (cs.cooling_system_fuel == HPXML::FuelTypeElectricity) && (cs.fraction_cool_load_served > 0) && !cs.is_shared_system } > 0)
+    args[:electric_panel_load_heat_pump_new_load] = !(existing_heating_electric && existing_cooling_electric) && (hpxml_bldg.heat_pumps.count { |hp| (hp.heat_pump_fuel == HPXML::FuelTypeElectricity) && !hp.is_shared_system } > 0)
```

Comment:
Wait, is this right? Shouldn't it be `!existing_heating_electric && !existing_cooling_electric && ...`? If you have a gas furnace w/ AC, your condition will evaluate to true, but it should be false. The HP is replacing an AC, so it's not a new load.